cloud_service/installing_cloud_ocp/verify-installation-cloud-ocp.adoc

```diff
@@ -9,14 +9,11 @@ toc::[]
 [role="_abstract"]
 After installing {product-title-managed-short}, you can perform some steps to verify that the installation was successful.
 
-:FeatureName: {product-title-managed-short}
-include::snippets/field-trial.adoc[leveloffset=+1]
-
 To verify installation, access your ACS Console from the {cloud-console}. The Dashboard will display the number of clusters that {product-title-managed-short} is monitoring, along with information about nodes, deployments, images, and violations.
```

Comment:
```suggestion
To verify installation, access your ACS Console from the {cloud-console}. The Dashboard displays the number of clusters that {product-title-managed-short} is monitoring, along with information about nodes, deployments, images, and violations.
```

Reply:
Made the first change. No article is needed for "information about nodes..." because it refers to _some_ of the available total information about nodes, deployments, images, and violations (not _all_ information).